src/app-layout/__tests__/main.test.tsx

```diff
@@ -169,29 +169,3 @@ test('a11y', async () => {
   );
   await expect(container).toValidateA11y();
 });
-
-test('drawers a11y', async () => {
```

Comment:
This test does not work anymore, because we now render drawer HTML, even when it is closed.

I do not think there is a way to make this test work while keeping the current implementation.

The real browser tests will cover this anyway: https://github.com/cloudscape-design/components/tree/main/src/__a11y__